### PR TITLE
Move docdock theme to submodule

### PR DESCRIPTION
**Pull request by renaissanceGeek**

### CODE REVIEW

File: docs-src/config.toml [hunk 1]
```diff
@@ -39,3 +39,6 @@ name = "<i class='fa fa-bookmark'></i> <label>Contact us</label>"
 identifier = "contactus"
 url = "https://www.cobaltspeech.com/contact"
 weight = 20
+
+[markup.goldmark.renderer]
+unsafe= true
```
Comment (mplooster) [inline docs-src/config.toml:44]: What does this mean? The word `unsafe` makes me worry, so I'm just checking that we feel "unsafe" is actually "safe enough".

**Review by mplooster — APPROVED** [review]
This is fantastic! Thanks for doing it!

**Review by jhollowayj — APPROVED** [review]
As long as it works and looks good, I don't have any objections.

File: docs-src/config.toml (hunk 1 above)
Comment (renaissanceGeek) [reply docs-src/config.toml:44]: "unsafe" means "allow HTML tags to be rendered as HTML, I know what I'm doing" ;-)

Comment (renaissanceGeek) [reply docs-src/config.toml:44]: Otherwise, all the < and > get escaped and show up verbatim

**Pull request merged**